problems/unsuccessful_products.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: unsuccessful-products
+stages:
+  - development
+short_description: About 30 to 45% of new products fail to deliver any meaningful financial return. There are many reasons why this happens. 
+tags:
+  - Unsuccessful Products
+  - market
+  - timing
+  - competition
+  - support
+keywords:
+  - market
+  - timing
+  - competition
+  - support
+  - price
+  - promotion
+---
+
+# Unsuccessful Product
+
+## TL;DR
+
+About 30 to 45% of new products fail to deliver any meaningful financial return. This typically happens due to a number of reasons – poor product / market fit, high price, failure to understand customer needs, fixing a non-existing problem, etc. 
+
+## Why Do Some Products Fail? 
+
+#### Poor Product Quality
+
+Obviously, a product, which is of poor quality, cannot be sold.
+
+#### Higher Price
+
+Another reason for the failure of certain products is the price factor. Higher production and distribution costs may lead to a higher price. Such a product cannot be sold in a market consisting of buyers willing to pay only lower range prices.
+
+#### Poor Timing
+
+It is important that a product (to be successful) is introduced in the market at the correct time. If it is introduced at an unsuitable time it may turn out to be a failure.
+
+#### Inherent Defect
+
+There may be an inherent defect in the product, which may affect its market potentialities. Such a product may not be preferred by the buyers even if the defect is fixed later.
+
+#### Big Competition
+
+In the case there are many similar products, the buyer will have many alternatives. In such a condition, the product can be unsuccessful.
+
+#### Lack of Promotional
+
+Popularizing the brand, particularly, in the introduction stage of a product is essential. Such a step will ensure repeated buying and bring long-term benefits. Failure to do so will be disastrous for the product.
+
+#### Poor Support
+
+The support quality is another important thing. Non-existent or poor support can ruin the product's reputation. 
+
+#### Failure to Understand Consumer Needs and Wants
+
+Blinded by their own visions, some companies ignore negative user feedback. Their products can easily fail to meet customers' needs and wants.
+
+## Zune Story
+
+Microsoft first introduced this portable media player in 2006, with several new generations of the device to follow. Zune faced these major challenges: 
```

Comment:
Microsoft first introduced this portable media player in 2006, with several new generations of the device to follow.

The Zune faced several major challenges: namely, inevitable comparisons to the iPod, which rules the portable media marketplace, and the fact that its software is only available for Windows (so far). 
In a financial report covering the fiscal quarter ending in December 2008, Microsoft said Zune revenues had decreased by 54%, or $100 million.

Reply:
Wording changed.